_posts/2024-08-28-datafusion-comet-0.2.0.md

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
should the header be here? just curios

Reply:
No, we don't have rat set up in the site repo. I guess because we don't release any software from this repo.